packages/aleo-types/src/address.rs

```diff
@@ -0,0 +1,70 @@
+use std::str::FromStr;
+
+use bech32::primitives::decode::CheckedHrpstring;
+use bech32::Bech32m;
+use error_stack::{ensure, Report, Result, ResultExt};
+use serde::{Deserialize, Serialize};
+
+use crate::Error;
+
+const ADDRESS_PREFIX: &str = "aleo";
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Address {
+    address: String,
+}
+
+impl Address {
+    pub fn address(&self) -> &str {
```

Comment:
I would call this `as_str()` as well if you need it. Otherwise, you'll have weird calls like 'address.address()'